quic-client/tests/quic_client.rs

```diff
@@ -7,14 +7,14 @@ mod tests {
         solana_net_utils::bind_to_localhost,
         solana_perf::packet::PacketBatch,
         solana_quic_client::nonblocking::quic_client::{
-            QuicClientCertificate, QuicLazyInitializedEndpoint,
+             QuicLazyInitializedEndpoint,
         },
         solana_sdk::{packet::PACKET_DATA_SIZE, signature::Keypair},
         solana_streamer::{
             quic::{QuicServerParams, SpawnServerResult},
             streamer::StakedNodes,
-            tls_certificates::new_dummy_x509_certificate,
         },
+        solana_tls_utils::{QuicClientCertificate,new_dummy_x509_certificate},
```

Comment:
space after 'QuicClientCertificate', can you do a "cargo fmt"

Reply:
thanks, cargo fmt done as suggested on all files.